tests/tests.js

```diff
@@ -1001,3 +1001,134 @@ describe('character classes', () => {
 	}
 });
 
+const TRANSFORM_U = { unicodeFlag: 'transform', unicodeSetFlag: 'transform' };
```

Comment:
I’m confused by this combination of settings, so just to clarify:

1. The grammar for the `u` flag does not change with the `v` flag proposal.
2. Only the `v` flag enables the new functionality.
3. The two flags cannot be combined.

Reply:
I updated the readme, hopefully it makes it clear. By default `v` is transformed to `u`, but you can set `unicodeFlag: 'transform'` to transform it even more. This is a sensible default, because now many browsers support `u` natively.

We also do something similar in Babel, where class fields are transformed to ES6 classes and then you can choose to transform them even more to ES5.